extensions/wikia/MonetizationModule/MonetizationModuleHelper.class.php

```diff
@@ -113,15 +113,21 @@ public function getMonetizationUnits( $params ) {
 		$cacheKey = $this->getMemcKey( $params );
 		$log->debug( "MonetizationModule: lookup with cache key: $cacheKey", $loggingParams );
```

Comment:
minor: move this comment into the `else` block below to keep it closer to the actual lookup. In the case this is a page specific request no cache lookup is done (in this invocation).